<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EyeTrackerWindowClass</class>
 <widget class="QMainWindow" name="EyeTrackerWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EyeTracker</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lGyte">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ClearPixel</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="pixmap">
     <pixmap>Images/gyte_logo_new.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bStartHangmanGame">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>150</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kozuka Gothic Pro B</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(8, 148, 255, 169);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Start Hangman Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bStartHeatmapApplication">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>481</x>
      <y>230</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kozuka Gothic Pro B</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(8, 148, 255, 169);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>Start Heatmap Application</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>20</width>
      <height>261</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>261</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kozuka Gothic Pr6N B</family>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Eye Tracker</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>410</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kozuka Gothic Pr6N B</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Any questions? &lt;a href=&quot;www.eyetrackerproject.wordpress.com&quot;&gt;Visit our website&lt;/a&gt;.</string>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bAbout">
    <property name="geometry">
     <rect>
      <x>606</x>
      <y>415</y>
      <width>81</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kozuka Gothic Pr6N B</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>About Us</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bHelp">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>415</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kozuka Gothic Pr6N B</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>251</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/EyeTracker/eye-tracking.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Images/eye-tracking.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bTestTracking">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>415</y>
      <width>145</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kozuka Gothic Pr6N B</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Test Eye Tracking</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>833</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
